crew

```diff
@@ -570,10 +570,11 @@ def download
       end
     end
 
-    system "#{CURL} --retry 3 -#{@verbose}#LC - --insecure \'#{url}\' --output #{filename}"
+    puts File.basename(filename)
```

Comment:
I take it this was a debug line you forgot to remove?

Reply:
Oh yes it was, thank you.